common/common.scss

```diff
@@ -1,6 +1,6 @@
 body:not(.category-header) {
   // hides banners based on outcome of shouldShow
-  .category-header-banner {
+  .category-title-header {
```

Comment:
This was a bug I discovered while adding system tests. Not ideal to use CSS to hide stuff but I don't want to change too much in this commit.